docs/en/metadata_oidc_op.rst

```diff
@@ -129,10 +129,10 @@ The EC of an OP MUST configure a metadata of type **"openid_provider"**, that MU
     - See `OpenID.Discovery#OP_Metadata`_. See signature :ref:`supported_algs`.
     - |spid-icon| |cieid-icon|
   * - **request_object_encryption_alg_values_supported**
-    - See `OpenID.Discovery#OP_Metadata`_. See key encryption :ref:`supported_algs`.
+    - Until otherwise indicated by AgID, this MUST NOT be included.
```

Comment:
this is not neutral considering CIE

can we rephrase?
can we adopt the section regarding the supported algs even as security/implementation considerations?